canonical-websites/jaas.ai
# Render Markdown descriptions

## Done

- Charms and bundles with descriptions containing markdown are now rendered correctly.

## QA

- Visit /u/cmars/mattermost/16
- The charm description markdown should render correctly.
- Visit a bundle with Markdown in the description (I couldn't find one, so I just hacked some markdown into the model in `_parse_bundle_data`).
- The bundle description markdown should render correctly.

## Details

- Fixes: #114.
